game/src/test/scala/ScalatraBootstrapSpec.scala

```diff
@@ -0,0 +1,34 @@
+import org.scalatest._
+
+import io.aigar.model._
+import io.aigar.game._
+
+class ScalatraBootstrapSpec extends FlatSpec with Matchers {
+  def withInMemDatabase(testCode: (TeamRepository) => Any) {
+    val teamRepository = new TeamRepository(None)
+    val team1 = teamRepository.createTeam(Team(None, "secret",  "team1", 20))
+    val team2 = teamRepository.createTeam(Team(None, "secret?", "team2", 10))
+    val team3 = teamRepository.createTeam(Team(None, "secret!", "team3", 30))
+
+    try{
+      testCode(teamRepository)
+    }
+    finally {
+      teamRepository.dropSchema
+      teamRepository.closeConnection
+    }
+  }
+
+  "Creating the application" should "create a game with the players from the team repository" in withInMemDatabase { (teamRepository) =>
+    ScalatraBootstrap.fixedTeamRepository = Some(teamRepository)
```

Comment:
After reading a bit:

> ScalaTest's normal approach for running suites of tests in parallel is to run different suites in parallel, but the tests of any one suite sequentially.

[SO answer](http://stackoverflow.com/a/31504359/395386)

Then it would be safe for the tests that use this mechanism (which *should* only be ctor tests in `ScalatraBootstrapSpec`). So I think we could leave it as-is (another approach that I saw was to disable parallel tests for *all* ScalaTest tests through `sbt` -- not a fan of this) and we should be careful to only use this in `ScalatraBootstrapSpec` (I added comments to specify that). 

Are you okay with that? @DrPandemic